velox/expression/tests/ExpressionVerifier.cpp

```diff
@@ -199,6 +209,18 @@ ResultOrError ExpressionVerifier::verify(
   return {makeRowVector(commonEvalResult[0]), nullptr};
 }
 
+void ExpressionVerifier::persistReproInfoIfNeeded(
+    const VectorPtr& inputVector,
+    std::vector<column_index_t> columnsToWrapInLazy,
+    const VectorPtr& resultVector,
+    const std::string& sql,
+    const std::vector<VectorPtr>& complexConstants) {
+  if (!options_.reproPersistPath.empty() && !options_.persistAndRunOnce) {
```

Comment:
nit: should we add a log statement saying skipping persistance since repro path empty or persist and run once specified ?